src/constants/categories.test.js

```diff
@@ -64,6 +65,11 @@ CATEGORIES.forEach((category) => {
     expect(heroExistsInCharts(category.heroIndicatorId, category.subCategories)).toBe(true);
   });
 
+  it(`${category.navName || category.name}: name (or navName) should be ${MAX_CATEGORY_NAV_TEXT_LENGTH} chars or shorter`, () => {
+    const name = category.navName || category.name;
```

Comment:
Minor. Really I just want to make one more comment...

Technically `const name = category.navName || category.name;` could go up two lines and be used inside the `it()` as well.

Reply:
Yes, this is true. I wanted to package it neatly together.